templates/README.md

```diff
@@ -0,0 +1,141 @@
+# Concordium Smart Contract Templates
+
+This repo helps you to get a new Concordium smart contract project up and running quickly by leveraging the pre-existing templates in this git repository. There are several smart contract templates available in this repository (currently a `default` template and a `cis2-nft` template). For generating the smart contracts from the templates, you need the `cargo-concordium` crate. `Cargo-concordium` can be installed as described in the following guide:
+
+[Cargo-concordium setup](https://developer.concordium.software/en/mainnet/smart-contracts/guides/setup-tools.html#setup-tools)
+
+You also need the `cargo-generate` crate. `Cargo-generate` can be installed by running the following command:
```

Comment:
```suggestion
You also need the `cargo-generate` tool. `Cargo-generate` can be installed by running the following command:
```

Reply:
Should we also mention at the start that this assumes they have installed `cargo` version at least ...?

Reply:
I added the following version info (cargo-generate might work with lower versions as well but I haven't tested it. I recommend the up-to-date version for cargo-generate):

- For generating the smart contracts from the templates, you need the `cargo-concordium` tool **version 2.2.0 or greater**.

- You also need the `cargo-generate` tool **(recommended version 0.16.0 or greater)**.

Reply:
Great. looks good.